ssl/statem/statem_clnt.c

```diff
@@ -1347,7 +1347,14 @@ static int set_client_ciphersuite(SSL *s, const unsigned char *cipherchars)
     if (s->hit && (s->session->cipher_id != c->id)) {
         if (SSL_IS_TLS13(s)) {
             const EVP_MD *md = ssl_md(s->ctx, c->algorithm2);
-
+           /*
+            * This code eliminates UB if in sslapitest.c at the line #4780
+            * clntsess->cipher = NULL;
+            */
+            if (s->session->cipher == NULL) {
```

Comment:
This is really an assert, i.e. `if (!ossl_assert(s->esssion->cipher != NULL)) {`

Reply:
removed comment and fixed to assert